src/pages/index.tsx

```diff
@@ -1,10 +1,39 @@
 import type { NextPage } from 'next';
+import { Button } from '../elements/button';
+import { Copy } from '../identity/copy';
+import { Heading1 } from '../identity/heading-1';
+import { Heading2 } from '../identity/heading-2';
+import { Page } from '../layouts/page';
 
 const Home: NextPage = () => {
   return (
-    <div>
-      <h1>Hello World</h1>
-    </div>
+    <Page>
+      <header>
+        <Heading1>Willkommen toller Mensch</Heading1>
+        <Copy>
+          Ich hab gehört du interessierst dich für Nachhaltigkeit. schön. du weisst nicht wie du das als Firma angehen
+          kannst? Hier findest du einen schnell Check / abschätzung über den Ausstoss deiner Firma. Danach werden dir
+          Möglichkeiten zur Verbesserung dieses Fussabrucks geliefert.
+        </Copy>
+      </header>
+      <main>
+        <div className="px-4 pt-8 md:mx-auto text-center">
+          <a href="/rechner">
```

Comment:
You'll need to use `next/link` for this :) 

https://nextjs.org/docs/api-reference/next/link